src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -349,15 +355,14 @@ const RuleDetails = styled('div')`
 
   & > span:nth-child(4n + 1),
   & > span:nth-child(4n + 2) {
-    background-color: ${p => p.theme.offWhite2};
+    background-color: ${p => p.theme.offWhite};
```

Comment:
This is to match the mocks more

![image](https://user-images.githubusercontent.com/1421724/79931698-7372ef00-8400-11ea-9249-7bcbd7c000a9.png)

Reply:
Maybe a bit contrasted but we don't have the #BetterGrays yet

Reply:
imo links should be blue, it's difficult to tell those are links

Reply:
I'll bring back the blue, sounds good

Reply:
Blue is back